.travis.yml

```diff
@@ -24,17 +24,17 @@ env:
   global:
     - JRUBY_OPTS="-J-Xmx1024m -Xcompile.invokedynamic=false -J-XX:+TieredCompilation -J-XX:TieredStopAtLevel=1 -J-noverify -Xcompile.mode=OFF"
     - NEO4J_BOLT_URL="bolt://localhost:7472"
-    - SEABOLT_LIB=./seabolt-1.7.4-Linux-ubuntu-16.04/usr/local/lib/libseabolt17.so
+    - BOLT_LIB=./seabolt-1.7.4-Linux-ubuntu-16.04/usr/local/lib/libseabolt17.so
   matrix:
-    - NEO4J_VERSION=enterprise-3.5.6
-    - NEO4J_VERSION=enterprise-3.4.14
-    - NEO4J_VERSION=community-3.5.6
+    - NEO4J_VERSION=enterprise-3.5.6 SEABOLT_LIB=$BOLT_LIB
```

Comment:
update neo4j and jruby versions in .travis.yml and README

Reply:
I can only update neo4j, ruby is at 9.2.7.0